src/v2i-hub/CARMACloudPlugin/src/CARMACloudPlugin.h

```diff
@@ -83,7 +83,12 @@ using namespace OpenAPI;
 
 
 namespace CARMACloudPlugin {
-
+	
+typedef enum acknowledgement_status {
+	acknowledgement_status__acknowledged 		= 1, //Acknowledged by CMV
+	acknowledgement_status__rejected 			= 2, //CMV cannot process the TCM message
+	acknowledgement_status__not_acknowledged 	= 3  //CMV does not respond at all within the v2xhub repeatedly broadcast time period
+} e_ack_status;
```

Comment:
Do you ever use this alias? If not what is the purpose. Not super familiar with C++ aliasing but the use of typedef and e_ack_status seems unnecessary since you never use the alias e_ack_status. Maybe there is something about typedef or aliasing I dont understand yet.